src/app/visual/skip-link/skip-link-demo.component.scss

```diff
@@ -0,0 +1,11 @@
+#skip-link-screenshot {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 60px;
+  width: 200px;
+}
+
+.spacer {
+  height: 5000px;
```

Comment:
Since this is a visual demo, I lean towards making it as realistic as possible. The demo was a little hard to understand on first glance. After I looked at the underlying code, I totally got it. However, I think we could avoid that confusion if there was content in the page. What would you say to adding content (even if its lorum ipsum) instead of a giant empty space of 5000px?